common/wforce-geoip2.cc

```diff
@@ -156,7 +155,7 @@ bool WFGeoIP2DB::lookupDataValue(const ComboAddress& address, const std::vector<
     }
     for (auto& i : path) {
       if (i != nullptr)
-        delete i;
+        free(i);
```

Comment:
this could already be done in previous commit instead of first doing it there and then changing it here.